app/controllers/api/storage.php

```diff
@@ -654,11 +653,6 @@
             ->setContext('bucket', $bucket)
         ;
 
-        $deletes
```

Comment:
Didn't we have this to avoid a conflict when a file with a similar ID was recently deleted?

Reply:
@eldadfux this event should already be triggered when the file is deleted, right?